openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -39,7 +39,7 @@ class DiscussionXBlock(XBlock, StudioEditableXBlockMixin, XmlParserMixin):
     discussion_id = String(scope=Scope.settings, default=UNIQUE_ID)
     display_name = String(
         display_name=_("Display Name"),
-        help=_("Display name for this component"),
+        help=_("The display name for this component"),
```

Comment:
Not sure how this appears. Is it correct/preferred that this particular help string unlike the others isn't a sentence (i.e. does not end with a period)?